ecl/eclagent/eclgraph.cpp

```diff
@@ -591,7 +591,8 @@ bool EclGraphElement::prepare(IAgentContext & agent, const byte * parentExtract,
             break;
         case TAKif:
             {
-                Owned<IHThorArg> helper = createHelper(agent, NULL);
+                assertex(!subgraph->owner);
```

Comment:
Logically it should be passing subgraph->owner.  I suspect the current bug was because someone cut and pasted from one of the branches where the pointer should be null (because those activities can only occur in the root graph).